plugins/modules/rdns.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c), name <email>, year
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: rdns
+short_description: Manage Hetzner's dedicated server reverse DNS entries
+author:
+  - WIP
+description:
+  - Manage Hetzner's dedicated server reverse DNS entries.
+seealso:
+  - name: Reverse DNS documentation
+    description: Hetzner's documentation on reverse DNS.
+    link: https://docs.hetzner.com/dns-console/dns/general/reverse-dns
+extends_documentation_fragment:
+- community.hrobot.robot
+
+options:
+  ip_address:
+    description: The IP address that should point to I(rdns).
+    type: str
+    required: yes
+  rdns:
+    description:
+      - The DNS address the I(ip_address) should resolve to.
+      - Omit the param to reset the reverse DNS entry to the default value.
```

Comment:
This parameter should be required when `state == present`. There should not be two different ways to delete an rDNS entry.